.github/workflows/publish-ghcr.yml

```diff
@@ -6,7 +6,7 @@ on:
       - main
 
 env:
-  CHARACTER_CONTAINER_APP_NAME: gm-toolset-character-api
```

Comment:
We already have a resource group set as gm-toolset. I think it makes more sense to have this path  to co container:
gm-toolset/character-api
Instead of
gm-toolset/gm-toolset-character-api